test/quantization/fx/test_quantize_pt2e.py

```diff
@@ -20,6 +20,7 @@
     QConfigMapping,
     observer,
 )
+from torch.ao.quantization.fx.prepare import _insert_observer
```

Comment:
why do we need this? this is an internal function that we may remove or change, I don't think it should be used in tests